code/modules/mob/living/simple_animal/animal_defense.dm

```diff
@@ -108,11 +108,14 @@
 	else
 		apply_damage(damage, damagetype, null, getarmor(null, armorcheck))
 		return TRUE
-
+/*
```

Comment:
If we're commenting out things, we should just remove them.

Reply:
This is the way we have been handling modular removals so far.